content/docs/introduction/high-availability.md

```diff
@@ -0,0 +1,100 @@
+
+- **Pageservers**
+
+  Pageservers act as a local disk cache, ingesting and indexing data from the WAL stored by Safekeepers and serving that data to your compute. In case of a Pageserver failure, data remains safe in S3, but impacted projects may become temporarily unavailable until reassigned to a new Pageserver by the Control Plane. For databases over 64 GB, this is an automatic process managed by the Storage Controller. For databases under 64 GB, this requires manual intervention on the Neon side (alerts and migration strategies).
+
+- **Object storage**
+
+  The primary, durable copy of your data resides in **cloud object storage** (e.g., S3), with **99.999999999%** durability to safeguard against any permanent data loss in even of Pageserver or Safekeeper failure.
+
+### Storage controller for automatic failover
+
+For larger databases (64 GB or more), Neon leverages the **Storage Controller** to handle **automatic failover** across Pageservers. The Storage Controller continuously monitors Pageservers using a heartbeat mechanism and automatically triggers failover when a Pageserver becomes unavailable. This process ensures that your database remains operational without manual intervention.
+
+Here’s a simplified view of how the Storage Controller manages Pageserver failover:
+
+![HA storage controller](/docs/introduction/HA-storage-controller.png)
+
+In this architecture:
+
+- **Storage Controller**: The Storage Controller monitors Pageservers through a heartbeat mechanism. When a Pageserver fails, the Storage Controller detects this failure and triggers the automatic failover process.
+- **Automatic failover**: Once the failure is detected, the Storage Controller reassigns the project to a healthy Pageserver, which uses the WAL stream from teh Safekeepers to reconstruct the state of the database. This failover is typically fast and happens without manual intervention.
+
+#### Sample failover
+
+The chart below shows the impact of a Pageserver failure (the truncated green line). When a Pageserver fails, the projects it was handled are automatically reassigned to other healthy Pageservers. This migration starts immediately, proceeds without human intervention, and typically completes within seconds. By the next metric scrape interval, the system is fully operational again.
+
+![chart showing pageserver failover via storage controller](/docs/introduction/HA-storage-failover-chart.png)
+
+## Limitations
+
+_No cross-region replication._ Neon's HA architecture is designed to mitigate failures within a single region by replicating data across multiple AZs. However, we currently do not support real-time replication across different cloud regions. This means that in the event of a region-wide outage, your data is not automatically replicated to another region and availability is limited to the scope of the affected region.
```

Comment:
I wonder if it would be good to make it clear here again that you're data is still safe. 

Also, "availability is limited to the scope of the affected region" is a little hard to grok. Perhaps simpler terms like "This means that in the event of a region-wide outage, your data is not automatically replicated to another region, and availability is dependent on the cloud provider restoring service to the affected region. However, such region-wide outages are extremely rare.